Shaders/SMAA.fx

```diff
@@ -43,6 +51,29 @@ uniform int CornerRounding <
 	ui_tooltip = "Determines the percent of anti-aliasing to apply to corners.";
 > = 0;
 
+#if SMAA_PREDICATION == 1
```

Comment:
I'd prefer if the check was just `#if SMAA_PREDICATION` or `#if SMAA_PREDICATION != 0` to follow standard practice on handling integer to boolean.